src/EventStore.ClientAPI/IEventStoreConnection.cs

```diff
@@ -49,111 +50,114 @@ public interface IEventStoreConnection : IDisposable
         Task<DeleteResult> DeleteStreamAsync(string stream, long expectedVersion, UserCredentials userCredentials = null);
 
         /// <summary>
-        /// Deletes a stream from the Event Store asynchronously
+        /// Deletes a stream from Event Store asynchronously.
         /// </summary>
         /// <param name="stream">The name of the stream to delete.</param>
         /// <param name="expectedVersion">The expected version that the streams should have when being deleted. <see cref="ExpectedVersion"/></param>
         /// <param name="hardDelete">Indicator for tombstoning vs soft-deleting the stream. Tombstoned streams can never be recreated. Soft-deleted streams
+        /// TODO: Can the below be a link?
```

Comment:
If you mean adding an external link, you can do that with an `<a>`, for example: `<a href="link">link text</a>`.
The link will show in intellisense, and docFx should be able to parse it when it's compiled.